### PR TITLE
Note that GaugeSummary supports all accessors of CounterSummary.

### PR DESCRIPTION
**Pull request by epgts**
# Description

Update for toolkit 1.7.0.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Comment by epgts** [comment]
We shouldn't merge this until it becomes true after we release 1.7.0.

**Review by charislam — APPROVED** [review]

**Comment by Loquacity** [comment]
> We shouldn't merge this until it becomes true after we release 1.7.0.

When is that due?

**Comment by epgts** [comment]
We're ready now. Thanks!

**Pull request merged**
